types/object-fit-images/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for object-fit-images 3.2
+// Project: https://github.com/bfred-it/object-fit-images#readme
+// Definitions by: Ranjit Singh <https://github.com/rann91>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function objectFillImages(
+    images: string | HTMLElement | HTMLElement[] | NodeList | null,
+    options?: { watchMQ: boolean },
+): void;
+
+export default objectFillImages;
```

Comment:
You should also think how this will be consumed, is that only expected to be consumed via ES6 module import?
It looks like browser loaded library with option to be consumed via es6 module, so e.g. could work AFAIK:
```ts
export as namespace objectFillImages;

export = objectFillImages;

declare function objectFillImages(
    images: string | HTMLElement | HTMLElement[] | NodeList | null,
    options?: { watchMQ?: boolean, skipTest?: boolean },
): void;
```